### PR TITLE
peter_task_action: 0.0.1-3 in 'indigo/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by sk2visea**
Increasing version of package(s) in repository `peter_task_action` to `0.0.1-3`:
- upstream repository: ssh://zhangwei@yunjiapp.com:29418/zhangwei/peter_task_action.git
- release repository: ssh://zhangwei@yunjiapp.com:29418/zhangwei/test_release.git
- distro file: `indigo/distribution.yaml`
- bloom version: `0.5.20`
- previous version for package: `null`
## peter_task_action

```
* change to small map mode
* 123
* 123
* gi:q
* 123
* 123
* 123
* 123
* 123
* 123
* 123
* 123
* 123
* sdf
* 123
* 123
* Update peter_task_action_multi_floor.cpp
* 123
* 123
* 123
* 123
* 123
* 123
* 123
* 123
* Update peter_task_action_multi_floor.cpp
* sfa
* dfa;
* 123
* Delete model.yaml
* 123
* Create param/model.yaml
* Delete task_status.yaml
* 123
* 123
* 123
* 123
* 123
* 123
* Update marker_map.yaml
* 撒旦飞洒的
* 123
* 123
* 111
* 123
* 123
* 123
* 123
* 123
* 123
* 123
* 123
* 1321
* 123
* 123
* 123
* 123
* 123
* 123
* 123
* 123
* 123
* 123
* 123
* 123
* 123
* 123
* 123
* 123
* 123
* 123
* 123
* 123
* Update peter_task_action_multi_floor.cpp
* 123
* 123
* 123
* 12
* 12
* 123
* Merge branch 'master' of ssh://yunjiapp.com:29418/zhangwei/peter_task_action
* 123
* 123
* Contributors: JustBen, Zhang Wei, zhangwei
```

### CODE REVIEW

File: indigo/distribution.yaml [hunk 1]
```diff
@@ -6476,6 +6476,12 @@ repositories:
       url: https://github.com/ros-perception/perception_pcl.git
       version: indigo-devel
     status: maintained
+  peter_task_action:
+    release:
+      tags:
+        release: release/indigo/{package}/{version}
+      url: ssh://zhangwei@yunjiapp.com:29418/zhangwei/test_release.git
```
Comment (dirk-thomas) [inline indigo/distribution.yaml]: You must use publically accessible URLs here.

**Comment by dirk-thomas** [comment]
You should also consider to update your changelog since the extracted commit messages in your repository are not helpful at all.

File: indigo/distribution.yaml (hunk 1 above)
Comment (dirk-thomas) [inline indigo/distribution.yaml]: I think the release repo must even be on github.com (as the error message in the travis log indicates).

**Comment by tfoote** [comment]
closing due to no response. I suggest you rerun the release on new publicly accessible repos to open a replacement pull request.

**Pull request closed without merging**